src/Core/Data/Mapping/YAMLKeys.hpp

```
@@ -105,3 +108,467 @@ namespace YAMLData {
     _YAMLStrType Render_MeshRenderable_MeshPath             = "MeshPath";
     _YAMLStrType Render_MeshRenderable_VisualScale          = "VisualScale";
 }
+
+
+// Descriptions for YAML keys
+namespace YAMLKeyDescription {
+    struct KeyDescription {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: description, unit, type

(cppcoreguidelines-pro-type-member-init)